conf/snapshot_clone_server.conf

```diff
@@ -35,7 +35,8 @@ server.snapshotPoolThreadNum=256
 # 快照后台线程扫描等待队列和工作队列的扫描周期(单位：ms)
 server.snapshotTaskManagerScanIntervalMs=1000
 # 转储chunk分片大小
-server.chunkSplitSize=1048576
+# for nos, pls set to 1048576
+server.chunkSplitSize=8388608
```

Comment:
@xu-chaojie pls check this, if we change the default value, we may encounter compatible problem when upgrading existing cluster.

Reply:
It is no problem